core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -1478,10 +1487,31 @@ public ServerBuilder contentPreview(int length) {
         return contentPreview(length, ArmeriaHttpUtil.HTTP_DEFAULT_CONTENT_CHARSET);
     }
 
+    /**
+     * Sets the {@link ExceptionHandlerFunction}s, {@link RequestConverterFunction}s
+     * and {@link ResponseConverterFunction} for creating a {@link AnnotatedHttpServiceExtensions}.
+     *
+     * @param exceptionHandlerFunctions the {@link ExceptionHandlerFunction}s
+     * @param requestConverterFunctions the {@link RequestConverterFunction}s
+     * @param responseConverterFunctions the {@link ResponseConverterFunction}s
+     */
+    public ServerBuilder annotatedHttpServiceExtensions(
+            Iterable<? extends ExceptionHandlerFunction> exceptionHandlerFunctions,
+            Iterable<? extends RequestConverterFunction> requestConverterFunctions,
+            Iterable<? extends ResponseConverterFunction> responseConverterFunctions) {
```

Comment:
Please reorder parameters: `requestConverterFunctions`, `responseConverterFunctions` and `exceptionHandlerFunctions`.